src/pages/joinGameLobby/[id].js

```diff
@@ -99,24 +119,48 @@ export default function Lobby() {
 
   useEffect(() => {}, [count]);
   useEffect(() => {
+    //if they are not Host
     if (getCookie("isHost") === "false") {
       setButtonDisabled(true);
+    } else{
+      //add a POST request to update users in lobby
     }
+
+    const urlCode = window.location.pathname.split('/');
+    console.log(urlCode[urlCode.length -1])
     setGameLobbyText(urlCode[urlCode.length -1])
+    getUserName();
   }, []);
 
   let copyClick = () => {
     navigator.clipboard.writeText(gameLobbyText);
     setCopy(true);
     setCopyWord("Copied !");
   };
+useEffect(() => {
+  if(userName !== undefined){
+    setTimeout(() => {
+      setInterval(function () {getUserNames(gameLobbyText)}, 5000);
+    }, 3000)
+  } else{
+    setTimeout(() => {
+      setInterval(function () {getUserNames(gameLobbyText)}, 5000);
+    }, 3000)
+  }
+}, [])
+
+
   return (
     <div style={containerStyle}>
       <div style={boxStyle}>
         <div style={titleStyle}>Lobby</div>
         <div style={contentStyle}>
           <div style={listStyle}>
             <h2 style={listHeaderStyle}>Player List</h2>
+            <button
+            onClick = {() => {if(getCookie('isHost') === 'false'){joinLobby()
+            }else{hostLobby()} }}
+            style = {{backgroundColor: "GREEN"}}>Ready Up</button>
```

Comment:
these two functions, joinLobby() and hostLobby(), would work better on the "join" and "host" buttons on the index.js file in this folder. The navigation to this page from there already works perfectly and the user has "user" in localstorage by that point, and then all this page needs to do is the "getLobby"ing that it's already doing to keep the list updated